src/main/java/rx/internal/operators/OperatorFinally.java

```diff
@@ -33,6 +33,9 @@
     final Action0 action;
 
     public OperatorFinally(Action0 action) {
+        if (action == null) {
```

Comment:
Saving one allocation does not matter in this case because in most of the cases app will crash after this exception :)